airgun/entities/virtwho_configure.py

```diff
@@ -48,6 +48,7 @@ def create(self, values):
         values = self._reset_values(values)
         view.fill(values)
         view.submit.click()
+        view.flash.assert_message(f"Success alert: Successfully created {values['name']}.")
         view.flash.assert_no_error()
```

Comment:
> Since we are verifying the success message in the flash, we can avoid this check as the intention of verifying the message is being repeated. Fail of `assert_message` check should raise a AssertionError aborting `assert_no_error` check.
> 
> Not a blocking comment though!

@jyejare This is because of the bug  https://bugzilla.redhat.com/show_bug.cgi?id=2179721
we should check the message display successfully and in the right format.

Reply:
Agree ! My comment is about the next line verification which is kinda duplicating! But as I said its not a blocker.